frontend/src/translations/zh-hant/common.json

```diff
@@ -71,7 +71,7 @@
       "no": "否"
     },
     "date_time": {
-      "date_time_format": "Modo, yyyy, aaah:mm:ss",
+      "date_time_format": "yyyy年M月d日, aaah:mm:ss",
```

Comment:
Is that ok that we have fixed values here instead of templates?
The documentation for formatting is here: https://date-fns.org/v2.29.3/docs/format

Reply:
Thanks for reminding. I tried formatting without fixed characters, but didn't find a way that works as expected last time.

I just tried again and find some examples here and it's working now: https://github.com/date-fns/date-fns/blob/main/src/locale/zh-CN/snapshot.md 😄